### DISCUSSION TITLE
Email Reports

### DISCUSSION THREAD

**Discussion by bhanuprakashd** [apache/superset#13994] [Q&A / Help]
I enabled email_reports everthing getting displayed but mail not going and following error is thrown
DEBUG:cron_descriptor.GetText:Failed to find locale English_India

**Comment by kamalkeshavani-aiinside** [comment]
```
DEBUG:cron_descriptor.GetText:Failed to find locale English_India
```
That is just a debug log and not an error. Have you enabled celery worker and celery beat? Can you see the execution log in celery and superset?

**Comment by cemremengu** [comment]
Any solutions or tips for this?
